drivers/ethernet/eth_adin2111.c

```diff
@@ -882,11 +1052,7 @@ static int adin2111_init(const struct device *dev)
 		return ret;
 	}
 
-#if CONFIG_ETH_ADIN2111_SPI_CFG0
 	val |= ADIN2111_CONFIG2_CRC_APPEND;
```

Comment:
The CRC is now always enabled.

Reply:
This is related to eth frame crc32, not connected to spi crc8

Reply:
Rechecked RM, indeed.